configuration/enable-offline-mode.adoc

```diff
@@ -46,14 +46,17 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner contains a local vulnerability definitions database. If {rh-rhacs-first} is running in normal mode, Scanner fetches new vulnerability definitions from Central. Central retrieves the updates from the internet, and then updates the Central database.
```

Comment:
> and then updates the Central database

Central does store the data, yes, but we care more about Scanner storing the data.

```suggestion
Scanner maintains a database of vulnerabilities which it fetches from Central. If {rh-rhacs-first} is running in normal mode, Central fetches the latest vulnerability data from the internet to pass back to Scanner.
```